index.html

```diff
@@ -42,5 +41,182 @@ <h5>LET'S CONNECT</h5>
+                </div>
+                <ul class=" works-flex-item titles-container">
+                    <li class="list-main">CANOPY</li>
+                    <li class="titles-item">Back End Dev</li>
+                    <li class="titles-item">2015</li>
+                </ul>
+                <div class="works-flex-item works-description">
+                   A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </div>
+                <div class="works-flex-item">
+                    <div class="lang-container">
+                        <div class="lang">html</div>
+                        <div class="lang">css</div>
+                        <div class="lang">javascript</div>
+                    </div>
+                   
+                </div>
+                <div class="works-flex-item">
+                    <div class="btn-see">
+                        <a href="#" class="btn-see-project">See project</a>
+                    </div>
+                </div>
+            </div>
+            <div class="works-flex">
+                <div class="works-flex-item work-image">
+                    <img src="assets/images/portfolio_snapshoot/snapshoot4.png" class="img-style" alt="tonic works" />
+                </div>
+                <div class="works-flex-item">
+                    <div class="intro-text-works">Multi-Post <br/> Stories</div>
+                </div>
+                <ul class=" works-flex-item titles-container">
+                    <li class="list-main">CANOPY</li>
+                    <li class="titles-item">Back End Dev</li>
+                    <li class="titles-item">2015</li>
+                </ul>
+                <div class="works-flex-item works-description">
+                   A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </div>
+                <div class="works-flex-item">
+                    <div class="lang-container">
+                        <div class="lang">html</div>
+                        <div class="lang">css</div>
+                        <div class="lang">javascript</div>
+                    </div>
+                   
+                </div>
+                <div class="works-flex-item">
+                    <div class="btn-see">
+                        <a href="#" class="btn-see-project">See project</a>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="aboutme">
+            <div class="aboutme-wrapper">
+                <div class="headline-container">
+                    <div class="intro-text-about">About<br> Myself</div>
+                    <div class="intro-message-about">Hello, I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
```

Comment:
- [ ] Hi, As I suggested earlier please try to follow correct HTML semantics. a `div` is just a container element that can store any type of content. In this case, we would like to use an element that signifies this is text simply textual content. Please kindly use a `p` tag here. 

something like this:
```suggestion
                    <p class="intro-message-about">Hello, I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```